seed/fieldtrials_testing_config_generator.py

```diff
@@ -50,10 +54,48 @@ def _get_variations_revision(date: str, branch: str) -> str:
     return output.rstrip().decode('utf-8')
 
 
-def _get_seed_data(seed_git_path: str, variations_revision: str):
+def _get_variations_seed_legacy(revision: str):
     seed_string = subprocess.check_output(
```

Comment:
this will raise if the file is not found.

Reply:
you won't get to the non-legacy part of the code seed generator if the exception is thrown here.

Reply:
fixed.